mindsdb/interfaces/database/integrations.py

```diff
@@ -424,6 +424,7 @@ def get_data_handler(self, name: str, case_sensitive: bool = False, connect=True
         """
         handler = self.handlers_cache.get(name)
         if handler is not None:
+            ctx.used_handlers.add(getattr(handler.__class__, "name", handler.__class__.__name__))
```

Comment:
shouldn't it be already exist in `used_handlers` because it was added in [line 501](https://github.com/mindsdb/mindsdb/pull/11730/files#diff-451a790f8d2f47f54ef2eac8d08806748f566c86f2437d088a428132684aee0fR501)?